website/docs/d/org_user.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_user"
+sidebar_current: "docs-vcd-datasource-org-user"
+description: |-
+  Provides a vCloud Director Organization user data source. This can be used to read organization users.
+---
+
+# vcd\_org\_user
+
+Provides a vCloud Director Org User data source. This can be used to read organization users, including org administrators.
+
+Supported in provider *v2.10+*
+
+
+## Example Usage
+
+```hcl
+data "vcd_org_user" "my-org-admin" {
+  org  = "my-org"
+  name = "my-org-admin"
+}
+
+data "vcd_org_user" "my-vapp-creator" {
+  org     = "my-org"
+  user_id = "urn:vcloud:user:c311eb35-6984-4d26-3ee9-0000deadbeef"
+}
+
+output "admin_user" {
+  value = data.vcd_org_user.my-org-admin
+}
+
+output "vapp_creator_user" {
+  value = data.vcd_org_user.my-vapp-creator
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to which the user belongs. Optional if defined at provider level.
+* `name` - (Optional) The name of the user. Required if `user_id` is not set
+* `user_id` - (Optional) The ID of the user. Required if `name` is not set
```

Comment:
Nit: dots are missing at the end of the two lines above.

Reply:
fixed